src/Extension/Criterion/CodeRunCriterion.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Extension.Criterion
+{
+    public class CodeRunCriterion
```

Comment:
"Run" isn't a word we're using in the public API surface. Can you align the naming to the existing terminology?

Reply:
Refactored. The word "run" does not exist in the repo anymore.